### PR TITLE
CASMTRIAGE-4052

### PR DESCRIPTION
**Pull request by cdelatte-hpe**
### Summary and Scope

- Fixes: CASMTRIAGE-4052
- Requires: DOCs change as well
- Relates to:

#### Issue Type

- Bugfix Pull Request
- Docs Pull Request 

This is a fix to cub_tool to utilize container image sha256 values for csm in family upgrades

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [ ] I tested this on internal system (if yes, please include results or a description of the test)
- [ ] I tested this on a vshasta system (if yes, please include results or a description of the test)
- [ ] I tested this on a vagrant environment (if yes, please include results or a description of the test)
 
### Idempotency
 

 The fix is making it more idempotent..

### CODE REVIEW

**Comment by cdelatte-hpe** [comment]
@heemstra I picked up your squashed changes and I am not sure why they are even showing in this commit.  They look to be correct based off your merge.

File: boxes/ncn-node-images/storage-ceph/provisioners/common/install.sh [hunk 1]
```diff
@@ -124,6 +124,10 @@ prefix = "registry.local"
 location = "localhost:5000"
 insecure = true
 
+[[registry]]
```
Comment (rustydb) [inline boxes/ncn-node-images/storage-ceph/provisioners/common/install.sh:127]: Doesn't this conflict with the registry on line 131?

**Review by bklei — APPROVED** [review]

File: boxes/ncn-node-images/storage-ceph/provisioners/common/install.sh (hunk 1 above)
Comment (cdelatte-hpe) [reply boxes/ncn-node-images/storage-ceph/provisioners/common/install.sh:127]: no..  it supported as "registry remapping" in the upstream docs.  also see upstream example https://github.com/containers/podman/blob/main/test/registries.conf

**Review by rustydb — APPROVED** [review]

**Review by heemstra — APPROVED** [review]

**Comment by rustydb** [comment]
Fast-forward Succeeded!

**Pull request merged**